hack/mirror-ose-images-into-cluster.sh

```diff
@@ -27,14 +27,52 @@ METERING_HADOOP_IMAGE_TAG="${METERING_HADOOP_IMAGE_TAG:-$OSE_IMAGE_TAG}"
 GHOSTUNNEL_IMAGE_TAG="${GHOSTUNNEL_IMAGE_TAG:-$OSE_IMAGE_TAG}"
 OAUTH_PROXY_IMAGE_TAG="${OAUTH_PROXY_IMAGE_TAG:-$OSE_IMAGE_TAG}"
 
-METERING_ANSIBLE_OPERATOR_SOURCE_IMAGE="${METERING_ANSIBLE_OPERATOR_SOURCE_IMAGE:-"$SOURCE_IMAGE_NAMESPACE/openshift-ose-metering-ansible-operator:$METERING_ANSIBLE_OPERATOR_IMAGE_TAG"}"
-METERING_REPORTING_OPERATOR_SOURCE_IMAGE="${METERING_REPORTING_OPERATOR_SOURCE_IMAGE:-"$SOURCE_IMAGE_NAMESPACE/openshift-ose-metering-reporting-operator:$METERING_REPORTING_OPERATOR_IMAGE_TAG"}"
-METERING_PRESTO_SOURCE_IMAGE="${METERING_PRESTO_SOURCE_IMAGE:-"$SOURCE_IMAGE_NAMESPACE/openshift-ose-metering-presto:$METERING_PRESTO_IMAGE_TAG"}"
-METERING_HIVE_SOURCE_IMAGE="${METERING_HIVE_SOURCE_IMAGE:-"$SOURCE_IMAGE_NAMESPACE/openshift-ose-metering-hive:$METERING_HIVE_IMAGE_TAG"}"
-METERING_HADOOP_SOURCE_IMAGE="${METERING_HADOOP_SOURCE_IMAGE:-"$SOURCE_IMAGE_NAMESPACE/openshift-ose-metering-hadoop:$METERING_HADOOP_IMAGE_TAG"}"
-GHOSTUNNEL_SOURCE_IMAGE="${GHOSTUNNEL_SOURCE_IMAGE:-"$SOURCE_IMAGE_NAMESPACE/openshift-ose-ghostunnel:$GHOSTUNNEL_IMAGE_TAG"}"
-OAUTH_PROXY_SOURCE_IMAGE="${OAUTH_PROXY_SOURCE_IMAGE:-"$SOURCE_IMAGE_NAMESPACE/openshift-ose-oauth-proxy:$OAUTH_PROXY_IMAGE_TAG"}"
+# if the _IMAGE variable is set for any component, we assume it's got the $SOURCE_REGISTY_PREFIX and lives in the $SOURCE_IMAGE_NAMESPACE, so we remove those values to get the image digest or image tag value (including the `@sha256:` or `:`.
```

Comment:
minor nit: missing the closing parenthesis and this could be a multi-line comment imo.